recipe/meta.yaml

```diff
@@ -11,19 +11,18 @@ source:
 
 build:
   skip: true  # [py<37]
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 3
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - {{ compiler('c') }}
   host:
     - pip
     - python
-    - setuptools >=45
-    - setuptools-scm >=6.0.1
+    - setuptools
+    - setuptools-scm 6.0.1
```

Comment:
Missing `toml`, see https://codeberg.org/miurahr/inflate64/src/commit/70700b5b98466a039b3b98aed4816959c09ea310/pyproject.toml#L56

Reply:
`toml`/`tomli` is already installed unconditionally by our `setuptools_scm` recipe. I see no need to include it in each package and polute the dependency graphs with sub-dependencies.